query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -248,8 +244,19 @@ case class PromQlRemoteExec(queryEndpoint: String,
     }
 
     // TODO: Handle stitching with verbose flag
-    // FIXME need to send and parse query stats in remote calls
-    QueryResult(id, resultSchema.get("stdval").get, rangeVectors, QueryStats(),
+    QueryResult(id, resultSchema.get("stdval").get, rangeVectors, readQueryStats(response.queryStats),
       if (response.partial.isDefined) response.partial.get else false, response.message)
   }
+
+  def readQueryStats(queryStatsResponse: Option[Seq[RemoteQueryStats]]): QueryStats = {
+    val queryStats = QueryStats()
+    if (queryStatsResponse.isDefined && queryStatsResponse.get.nonEmpty) {
+      queryStatsResponse.get.foreach(stat => {
```

Comment:
remove extra parens, which is a java syntax  🙂

Reply:
This is taken care of.

Reply:
This is taken care of.